client/my-sites/importer/importing-pane.jsx

```diff
@@ -134,7 +134,7 @@ class ImportingPane extends React.PureComponent {
 
 		if ( pageCount && postCount ) {
 			return this.props.translate(
-				'All done! Check out {{a}}Posts{{/a}} or ' +
+				'All done! Check out {{a}}Posts{{/a}} and ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'All done! Check out {{a}}Posts{{/a}} or {{b}}Pages{{/b}} to see your imported content.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).